purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -8,11 +10,12 @@ import java.net.URL
  *
  * @see [Paywalls Documentation](https://rev.cat/paywalls)
  */
+@Serializable
 data class PaywallData(
     /**
      * The type of template used to display this paywall.
      */
-    val templateName: String,
+    @SerialName("template_name") val templateName: String,
```

Comment:
I was thinking most fields need to add this to support the snake_case to camelCase transformation and it would be good to do this automatically... But from a quick search, looks like there is [JsonNamingStrategy](https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/-json-naming-strategy/-builtins/-snake-case.html) but it's still experimental... So this might be the best for now...

Reply:
Yes! Thankfully it looks like they will add it soon